toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -65,7 +65,7 @@ func main() {
 	var toolchainPackages []string
 	toolchainManifest := *toolchainManifest
 	if len(toolchainManifest) > 0 {
-		toolchainPackages, err = schedulerutils.ReadReservedFilesList(toolchainManifest)
+		toolchainPackages, err = schedulerutils.ReadReservedPackageManifest(toolchainManifest)
 		if err != nil {
 			logger.Log.Fatalf("unable to read toolchain manifest file '%s': %s", toolchainManifest, err)
 		}
```

Comment:
Could we also extract this logic into a separate function and remove the duplication from this file and `scheduler.go`? I think we're doing the exact same thing in both, so we could just do `toolchainPackages := schedulerutils.ReadReservedPackageManifest(...)`. I'm guessing all we'd have to do, is to allow `schedulerutils.ReadReservedPackageManifest` to accept an empty string as input.

Reply:
Sure, that seems reasonable, will update.